### DISCUSSION TITLE
How can I search slashing/punishment details

### DISCUSSION THREAD

**Discussion by PerrywLiu** [filecoin-project/lotus#7841] [General Q&A] [answered]
I find I was punished on browser, How can I search punish deatil or other way?

**Comment by LexLuthr** [comment, marked as answer]
@PerrywLiu We can check it in the mine logs. But I don't know any other method.
Asking community to provide input on this.

**Reply by LexLuthr** [reply]
@PerrywLiu As discussed in the Slack, https://filecoinproject.slack.com/archives/CPFTWMY7N/p1641377351247500 there is no other way to track slashing except logs and wallet balances.
